lib/bindings/HTTPBindings.js

```diff
@@ -44,7 +44,7 @@ var http = require('http'),
     },
     transport = 'HTTP';
 
-function handleError(error, req, res, next) {
+function handleError(error, req, res) {
```

Comment:
Not sure if the fix for this is removing next from the signature of the function of this change is actually raising another kind of bug in the function: it should call next() upon completion but it is not doing it.

A similar case would be the one with returnCommands() modification, some lines below.

@AlvaroVega what do you think?

Reply:
I guess express is expecting always a function with req, res, next) args

Reply:
I have been discussing this case with @AlvaroVega and the solution here shouldn't be to remove "next" for the parametrs, but adding the `next();` call to the end of the function, i.e. just after L59.

Reply:
fixed in a2e11f62c9516932f9f4e42afc7a52b915322831